src/main/kotlin/org/rust/lang/core/psi/ext/RsDocAndAttributeOwner.kt

```diff
@@ -177,6 +182,11 @@ class QueryAttributes(
      */
     private fun attrsByName(name: String): Sequence<RsMetaItem> = metaItems.filter { it.name == name }
 
+    /**
+     * Get a sequence of all attributes matching the given [regex]
+     */
+    private fun attrsByRegex(regex: Regex): Sequence<RsMetaItem> = metaItems.filter { it.name?.let { name -> regex.containsMatchIn(name) } ?: false }
```

Comment:
I'd prefer something like
```kotlin
private fun attrsByName(regex: Regex): Sequence<RsMetaItem> = metaItems.filter {
    val name = it.name ?: return@filter false
    regex.matches(name)
}
```
because:
* we still match only name
* we don't want to match some substring inside attribute `name`
* it's more readable

Reply:
BTW, it doesn't work with `tokio::test` attribute because its name is null.
What's why the corresponding test fails